upgrade/helpers/tasks.py

```diff
@@ -771,30 +773,46 @@ def upgrade_using_foreman_maintain():
             run('foreman-maintain upgrade check --target-version {}'
                 ' -y'.format(os.environ.get('TO_VERSION')))
 
-    # whitelist disk-performance check
-    if os.environ.get('FROM_VERSION') == os.environ.get('TO_VERSION'):
-        # z stream upgrade
-        run('foreman-maintain upgrade run '
-            '--whitelist="disk-performance" '
-            '--target-version {} '
-            '-y'.format(os.environ.get('TO_VERSION') + ".z"))
-    else:
-        # use beta until 6.7 is GA
-        if os.environ.get('TO_VERSION') == '6.7':
-            with shell_env(FOREMAN_MAINTAIN_USE_BETA='1'):
-                if bz_bug_is_open(1781128):
-                    run('foreman-maintain upgrade run '
-                        '--whitelist="disk-performance, foreman-tasks-not-paused" '
-                        '--target-version {} -y'.format(os.environ.get('TO_VERSION')))
-                else:
-                    run('foreman-maintain upgrade run '
-                        '--whitelist="disk-performance" '
-                        '--target-version {} -y'.format(os.environ.get('TO_VERSION')))
-        else:
+    def satellite_upgrade():
+        """ This inner function is used to perform Y & Z satellite stream upgrade"""
+        # whitelist disk-performance check
+        if os.environ.get('FROM_VERSION') == os.environ.get('TO_VERSION'):
+            # z stream satellite upgrade
             run('foreman-maintain upgrade run '
                 '--whitelist="disk-performance" '
+                '--target-version {} '
+                '-y'.format(os.environ.get('TO_VERSION') + ".z"))
+        else:
+            # use beta until 6.7 is GA
+            if os.environ.get('TO_VERSION') == '6.7':
```

Comment:
```suggestion
            if os.environ.get('TO_VERSION') == '6.8':
```
This should be 6.8 right?

Reply:
Thanks, Jameer, It should be 6.8.